### DISCUSSION TITLE
Client Login - VMDK image

### DISCUSSION THREAD

**Discussion by seancarte** [ltsp/ltsp#230] [Q&A]
I'm not able to log in when using a vmdk image. The error is: '.Login incorrect.'

I have a user 'opac' set to autologin in the VirtualBox image, and I can log in with the password there too.

I have the user defined on the server also, with the same password. I ran `ltsp initrd`. I can log in with this user if I use the chrootless image.

Prior to creating the 'opac' user on the server, when attempting to log in using the vmdk image, the error I received was 'User opac doesn't exist'. So it does seem like authentication relies on users defined on the server.

I have tried logging in without the GUI (Ctrl-F1). The error there is:
/usr/share/ltsp/client/login/pamltsp: 181: /usr/share/ltsp/client/login/pamltsp: sshfs: not found
.Login incorrect.

Line 181 in pamltsp is:
sshfs -o "$sshfs_params" "$@" "$pw_name@$SERVER:" "$pw_dir"

I don't see anything helpful in syslog or authlog.

**Comment by alkisg** [comment]
Inside the VMDK image, you're supposed to install the "ltsp" package, which will also install sshfs for you. Then run `ltsp image` and test again.

**Comment by alkisg** [comment]
P.S. as you figured out, users should be created on the server, not inside the VM.

**Comment by seancarte** [comment]
In the VM:

`add-apt-repository ppa:ltsp`
`apt update`
`apt install ltsp`

... and it worked!

Thank you, @alkisg !

The next challenge will be to get auto-login working ...
